src/components/HomePageLogo.tsx

```diff
@@ -1,12 +1,28 @@
-import React from 'react';
+import React, { JSX } from 'react';
+import { ThemeDataType, ThemeType } from 'src/themes';
+import { useTheme } from 'styled-components';
 import { LogoContainer } from './HomePageLogo.style';
 import { ArDriveLogoLight } from './icons/ArDriveLogoLight';
+import { ArDriveLogoDark } from './icons/ArDriveLogoDark';
 
 export function HomePageLogo(): JSX.Element {
+	const themeData = useTheme() as unknown as ThemeDataType;
+	const { isEmbedded } = themeData;
+	const { themeName: current } = themeData;
```

Comment:
Can combine to be `{isEmbedded, themeName: current}`. Also, why rename to current? It's a bit confusing since there's also a current field in the themeData. Would modify code to:

```
	const { isEmbedded, themeName } = useTheme();
	const style: React.CSSProperties = {
		display: isEmbedded ? 'none' : 'block'
	};
	const logo =
		themeName === ThemeType.light ? (
			<ArDriveLogoLight />
		) : (
			<ArDriveLogoDark />
		);
		```

Reply:
Applied - 2a39310a2a5b6fda2f627c2ed38afcca0d36cf81